### PR TITLE
Update step-by-step tutorial

### PR DESCRIPTION
**Pull request by Kjili**
Update phrasing for a better flow in reading, minor grammar fixes. Removes parts I stumbled over or got stuck at while reading the tutorial.

### CODE REVIEW

**Comment by StraToN** [comment]
I'm fine with this commit. I'll let @akien-mga look at it as well.

File: tutorials/step_by_step/animations.rst [hunk 1]
```diff
@@ -10,14 +10,14 @@ This tutorial will explain how everything is animated in Godot. Godot's
 animation system is extremely powerful and flexible.
 
 To begin, let's just use the scene from the previous tutorial (:ref:`doc_splash_screen`).
-The goal will be to add a simple animation to it. Here's a copy
+The goal is, to add a simple animation to it. Here's a copy
```
Comment (akien-mga) [inline tutorials/step_by_step/animations.rst]: Why a comma after the verb?

File: tutorials/step_by_step/animations.rst [hunk 2]
```diff
@@ -47,8 +47,8 @@ for similar functionality.)
 
 .. image:: /img/animeditor.png
 
-But the second, and most important, is that the property editor enters
-into "animation editing" mode. In this mode, a key icon appears next to
+The second and most important, is that the property editor enters
```
Comment (akien-mga) [inline tutorials/step_by_step/animations.rst]: If you remove the first comma, you need to remove the second. Now the sentence is cut in a grammatically incorrect way.

File: tutorials/step_by_step/filesystem.rst [hunk 3]
```diff
@@ -46,7 +46,7 @@ engine.cfg
 ----------
 
 The engine.cfg file is the project description file, and it is always found at
-the root of the project, in fact it's location defines where the root is. This
+the root of the project. In fact it's location defines where the root is. This
```
Comment (akien-mga) [inline tutorials/step_by_step/filesystem.rst]: While fixing English mistakes, it would have been nice to fix this hideous `it's` too :P

**Comment by akien-mga** [comment]
Sorry for the delay. The changes are quite good :)

**Pull request merged**